system_tests/monitoring.py

```diff
@@ -201,18 +202,24 @@ def test_write_point(self):
 
         retry_500(client.write_point)(metric, resource, VALUE)
 
-        def _query_timeseries_with_retries():
-            MAX_RETRIES = 7
+        MAX_RETRIES = 7
+        
+        # need to wrap built-in function for decorators to work
+        def list_timeseries(query):
+            return list(query)
 
-            def _has_timeseries(result):
-                return len(list(result)) > 0
+        def _has_timeseries(results):
+            return len(results) > 0
 
-            retry_result = RetryResult(_has_timeseries,
-                                       max_tries=MAX_RETRIES)(client.query)
-            return RetryErrors(BadRequest, max_tries=MAX_RETRIES)(retry_result)
+        endtime = datetime.datetime.utcnow()
+        endtime += datetime.timedelta(minutes=1)
```

Comment:
This line is required, this is the main issue I'm trying to fix in relation to #2459.